app.js

```diff
@@ -23,6 +24,8 @@ app.use((req, res, next) => {
 })
 
 app.use(cors())
+app.use('/email', express.json({ limit: '10mb' }))
+app.use('/email', emailRouter)
```

Comment:
Can you move these two lines to where we define other routers below?

Reply:
it doesn't not work if the route is after the "app.use(express.json())", it think it must be implemented before or it throws the payload too large error. I tried moving it and got the payload too large error even if the file size was 500kb